build.proj

```diff
@@ -23,6 +23,7 @@
     <Project Include="external\dir.proj" />
     <Project Include="src\ref.builds" />
     <Project Include="src\src.builds" />
+    <Project Include="src\tests.builds" />
```

Comment:
agreed I would like to make the test build and running a separate step from build.cmd.